src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="true">
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" 
+      rel="stylesheet">
+  </head>
+  <body>
+    <div  class="block" data-qa="card">
+      <img
+        class="photo"
+        src="images/imac.jpeg"
+        alt="Content image"
+      >
+      <h1 class="discription">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="code">Product code: 195434</p>
+      <div class="rev">
+        <div class="star">
+          <div class="stars"></div>
```

Comment:
Use BEM. The container must have a name in the plural, and elements - in the singular.

[More info.](https://scalablecss.com/bem-quickstart-guide/)

Reply:
what should be done here? rename div to rev or make additional wrappers?